### DISCUSSION TITLE
"Merge databases..." v "Synchronise databases..."

### DISCUSSION THREAD

**Discussion by flagellatedcucumber** [keepassxreboot/keepassxc#6327] [Q&A] [answered]
In Windows/Keepass, File > Synchronise Databases... enables two kbdx files to synchronise, i.e. two-way synchronisation, resulting in two identical files, each a union of each others' entries.

In Linux/KeepassXC, File > Merge databases... does one-way synchronisation.

The user can run a merge twice, in both directions, which is fine, but have I missed a one-stop-shop command to synchronise two databases?

If there isn't one, is it worth a feature request, to mirror Keepass/Windows?

**Comment by droidmonkey** [comment, marked as answer]
Already requested

**Comment by CaptainBalou** [comment]
Hello @taxedserf and @droidmonkey ,

I was on my way opening an issue report but as I understand that here this is not a bug? Or have I misunderstood? @droidmonkey , where have you requested it? Is there a way to see when this feature is planned to be integrated (if so)?

All the best, stay healthy.
Marco

**Reply by droidmonkey** [reply]
Search the issues list please
